src/plugins/timelion/public/panels/timechart/schema.ts

```diff
@@ -349,6 +348,7 @@ export function timechartFn(dependencies: TimelionVisualizationDependencies) {
           if (options.yaxes) {
             options.yaxes.forEach((yaxis: any) => {
               if (yaxis && yaxis.units) {
+                const formatters = tickFormatters(yaxis) as any;
```

Comment:
it looks like this line will cause a problem in the `Timeline app`. Because I can't see where we changed the corresponding code for the `Timeline app`.